react/features/base/media/components/native/Video.js

```diff
@@ -45,7 +52,12 @@ export default class Video extends Component<*> {
          * values: 0 for the remote video(s) which appear in the background, and
          * 1 for the local video(s) which appear above the remote video(s).
          */
-        zOrder: PropTypes.number
+        zOrder: PropTypes.number,
+
+        /**
+         * Indicates if zooming (pinch & zoom and/or drag) is enabled or not.
+         */
+        zoomEnabled: PropTypes.bool
```

Comment:
maybe instead of making this feature specific we could just expose the 'objectFit' property and pass it over to the RTCView ? make the decision on the upper level